<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraChangeQuickPropertyForm</class>
 <widget class="GUIFormBase" name="CameraChangeQuickPropertyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="toolButtonDown">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>33</y>
     <width>65</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>▼</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonUp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>33</y>
     <width>65</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>▲</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxValue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>33</y>
     <width>74</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>明るさ</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonAdjustment">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>補正運転</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
